web_api/__init__.py

```
@@ -0,0 +1,319 @@
+        self.profile_store = profile_store
+        self.config_manager = config_manager
+        self.conversation_ledger = conversation_ledger
+        self.chat_sources = chat_sources
+        self.status_transition_manager = status_transition_manager
+        self.debounce_manager = debounce_manager
+        self.last_decisions = last_decisions
+
+    # ---------- 模板 ----------
+
+    async def list_profiles(self):
+        """模板列表 + 绑定总览（模板含完整 config）。
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)